server/container/core/src/test/java/org/apache/james/core/MailImplTest.java

```diff
@@ -18,37 +18,34 @@
  ****************************************************************/
 package org.apache.james.core;
 
-import org.apache.mailet.base.test.MailUtil;
-import org.apache.mailet.MailAddress;
-import org.apache.mailet.Mail;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.ByteArrayInputStream;
+import java.util.ArrayList;
 
 import javax.mail.MessagingException;
 import javax.mail.Session;
 import javax.mail.internet.MimeMessage;
 
-import java.io.ByteArrayInputStream;
-import java.util.ArrayList;
-import static org.junit.Assert.*;
+import org.apache.mailet.Mail;
+import org.apache.mailet.MailAddress;
+import org.apache.mailet.base.test.MailUtil;
 import org.junit.Test;
 
-public class MailImplTest extends MailTestAllImplementations {
+public class MailImplTest {
 
     private static final Session NO_SESSION = null;
-    
-    @Override
-    protected Mail createMailImplementation() {
-        return new MailImpl();
-    }
 
     @Test
     public void testConstr1() throws MessagingException {
         MailImpl mail = new MailImpl();
 
         helperTestInitialState(mail);
         helperTestMessageSize(mail, 0); // MimeMessageWrapper default is 0
-        assertNull("no initial message", mail.getMessage());
-        assertNull("no initial sender", mail.getSender());
-        assertNull("no initial name", mail.getName());
+
+        assertThat(mail.getMessage()).as("Checking there is an initial message").isNull();
```

Comment:
Don't you mean the contrary "Checking there is No initial message".
Same for the others lines?